docs/installation.rst

```diff
@@ -52,19 +53,17 @@ and python versions in the URL.
 .. code-block:: bash
 
     pip install \
-     apache-airflow==1.10.10 \
-     --constraint \
-            https://raw.githubusercontent.com/apache/airflow/1.10.10/requirements/requirements-python3.7.txt
+     apache-airflow==1.10.12 \
+     --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-1.10.12/constraints-3.7.txt"
```

Comment:
This is a broken link.

Correct link
```
https://raw.githubusercontent.com/apache/airflow/constraints-1-10/constraints-3.7.txt
```
But does that mean 1.10.10 and 1.10.11 have the some constraint file?

Reply:
It won't be broken when we release 1.10.12 and tag it.

Good point with 1.10.10 == 1.10.11  @kaxil . I thought they will work anyway, but we can do better easily.

Currently it is the same I just added one commit in the orphaned branch  and I added all tags there. But there is nothing to take the "frozen" requirements from 1.10.10 and add extra commit and tag it. I will do it momentariily.

Reply:
Now - I recovered 1.10.10 and 1.10.11 and they are separate commits : 

```
commit f53b1d0041d3dc087ff2c6addaa72266e784b928 (HEAD -> constraints-1-10, tag: constraints-1.10.11, apache/constraints-1-10, apache-ssh/constraints-1-10)
Author: Jarek Potiuk <jarek@potiuk.com>
Date:   Mon Jul 20 13:14:27 2020 +0200

    Added constraints for 1.10.11

commit 1c2d5de877a993a6541e017d4d59938315814448 (tag: constraints-1.10.9, tag: constraints-1.10.8, tag: constraints-1.10.7, tag: constraints-1.10.6, tag: constraints-1.10.5, tag: constraints-1.10.4, tag: constraints-1.10.3, tag: constraints-1.10.2, tag: constraints-1.10.10, tag: constraints-1.10.1)
Author: Jarek Potiuk <jarek@potiuk.com>
Date:   Sun Jul 19 19:35:22 2020 +0200

    Add orphan constraints for 1-10 branch 1.10.10 version


```